cachi2/core/models/envfile.py

```diff
@@ -0,0 +1,51 @@
+import json
```

Comment:
not sure if `core.models.envfile` is the right place for this, WDYT?

Reply:
This is a good question. I'm assuming we'll leave models for "struct" type of classes, with little actual behavior on them.

In that case, I think we could go for python packages that focus on specific behaviors, such as:

```
- core
  - content_manifest
     - cyclonedx.py
     - purl.py   
  - helpers
     - generate_env.py
     - generate_tarball.py
  - models
     - input.py
     - output.py
     - validators.py 
  - package_managers
     - gomod.py
     - pip.py
     - npm.py 
  - source_code
     - git.py 
```

Of course, this is just a brainstorming of names, we need to think them through. I just wanted to illustrate the approach.

Reply:
Yeah, that's why I was unsure about making it a model. I do want to keep the `EnvFormat` enum in the same module that generates the file content though

Reply:
So, are we changing this in this PR, or are we leaving it for a future one?

Reply:
This one, if I can find a sensible name for the module :cry: 

I kinda wanna just put it in `core.envfile`

Reply:
I came up `core.extras.envfile`, as in "useful stuff that's not part of the default fetch-deps functionality"

Although core.extras might sound a bit contradictory :smile:

Reply:
I suggested "helpers", but I think that helpers tend to become a box to "stuff I don't know where to put". But then, I think so does extras. :laughing: 

I'm fine with either one.

Reply:
Same here. Both sounds OK. :)